package.json

```
@@ -6,7 +6,8 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "lint": "eslint . --ext ts,tsx,js,jsx"
+    "lint": "eslint . --ext ts,tsx,js,jsx",
+    "add-member": "npx hygen gen intro"
```

🥳  🌟 🌟 
```suggestion
    "add-member": "hygen gen intro"
```

依存パッケージに hygen を追加していただけますか？
`npm i -D hygen`

---

基本 人によってですが一回しか使わないので 依存関係として入れたくないなぁと思ってます。
なので npx で 起動してまする

---

なるほど、、
最低限バージョンだけ指定した方がいいかなと思うんですが、どうでしょ？
(更新する人がnpmに明るい人とは限らないので)

`npx hygen@4’

---

>最低限バージョンだけ指定した方がいいかなと思うんですが、どうでしょ？

あー、動作保証してる最低限のバージョン...

そしたら依存関係に入れたほうが良さそうですね。 短慮でした。ご指摘ありがたいっす。
いれますね　

---

c1fb70146ea2ae0632ac1a34312ccb9731 で対応